praesentia/templates/home.html

```diff
@@ -50,6 +50,23 @@ <h1>Praesentia</h1>
                         <label>QR Image:</label>
                         <input type="file" accept="image/*" id="qrImageFile">
                     </div>
+                    <div class="shareScreen" id="qrInput">
```

Comment:
I think this can be made simpler. I am thinking about making a select box for selecting a QR scanning method. The select box will have only two options: image file and screen sharing. For each selected option, the UI is adjusted. I also think that the auto-submit feature should be removed (or just moved and adjusted so the feature can also be usable when the scanning method is not screen sharing).

What do you think?

Reply:
I agree with you, but I still can't use vanilla CSS, so I can't help you redesign. But for auto-submit, I agree to move it and adjust it because when I made this feature, I thought will use it to AFK while joining the meeting room🤣

Reply:
I see. I will wire it up in another PR. I am resolving this conversation.